### ISSUE TITLE
Please tag releases

### ISSUE THREAD

**Issue by szepeviktor** [pablo-pacheco/wp-namespace-autoloader#14]
Currently Composer errors out

> Could not find a version of package pablo-sg-pacheco/wp-namespace-autoloader matching your minimum-stability (stable).

**Comment by BrianHenryIE** [comment]
I agree this would be nice. For reference, I use this in my `composer.json`:
```
  "minimum-stability": "dev",
  "prefer-stable": true,
...
  "require": {
    "pablo-sg-pacheco/wp-namespace-autoloader": "dev-master",
```

  ~~"repositories": [
    {
      "url": "https://github.com/pablo-sg-pacheco/wp-namespace-autoloader",
      "type": "git"
    },~~

(looks like that's already mentioned in the README!)

**Comment by szepeviktor** [comment]
> ` "minimum-stability": "dev",`

This is not the friend of those involved in Production - e.g. me :)

**Comment by szepeviktor** [comment]
>   "repositories": [

This package is already published on Packagist 🎉 
https://packagist.org/packages/pablo-sg-pacheco/wp-namespace-autoloader

**Comment by BrianHenryIE** [comment]
Thanks. I knew it was on Packagist but I guess after I had been using my own fork in my `repositories` list, I wasn't sure I could use `dev-master` of the primary repo through Packagist.
